org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/copyqualifiedname/CopyQualifiedNameService.xtend

```diff
@@ -7,14 +7,12 @@
  *******************************************************************************/
 package org.eclipse.xtext.ui.editor.copyqualifiedname
 
-import org.eclipse.emf.ecore.EObject
-
 /**
  * @author Anton Kosyakov - Initial contribution and API
  * @since 2.4
+ * @deprecated Use org.eclipse.xtext.naming.CopyQualifiedNameService instead
  */
-interface CopyQualifiedNameService {
-
-	def abstract String getQualifiedName(EObject selectedElement, EObject context)
+@Deprecated
+interface CopyQualifiedNameService extends org.eclipse.xtext.naming.CopyQualifiedNameService {
```

Comment:
It was decided that the interface will be prefixed with "I"

Reply:
Done.